puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/wrappers/abort_build.yaml

```diff
@@ -0,0 +1,8 @@
+- wrapper:
+    name: abort_build
+    wrappers:
+      - timeout:
+          type: absolute
+          timeout: {timeout}
```

Comment:
I have tried a number of things and so far this just isn't populating. Going to continue investigating but I am bit stumped based upon the documentation.